global.json

```diff
@@ -1,14 +1,14 @@
 {
   "sdk": {
-    "version": "8.0.100",
+    "version": "10.0.100-preview.6.25302.104",
```

Comment:
This would force us to install P6 on every machine that we use for NuGet.Client, but we don't have a real dependency to P6.

Can we not do this change?
I think bumping to 9.0.100 is actually needed/correct, but 10 p6 is a bit too fast to enforce to everyone right now.